src/JitInterface/src/CorInfoImpl.cs

```diff
@@ -1865,12 +1864,15 @@ private void getCallInfo(ref CORINFO_RESOLVED_TOKEN pResolvedToken, CORINFO_RESO
 
             pResult._exactContextNeedsRuntimeLookup = 0;
 
-            // TODO: CORINFO_VIRTUALCALL_STUB
```

Comment:
It is redundant with `TODO: Interface methods`. We may or may not choose to use VirtualCall_Stub for them, depending on how exactly we wire the interface methods in.